CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.13.8
+
+<!-- generated comparing v5.13.7..master -->
+
+_Jul 11, 2023_
+
+A big thanks to the 15 contributors who made this release possible. Here are some highlights ✨:
+
+- 💫 Exports from Material UI, Base UI and Joy UI are now prepended with the `"use client"` directive (#37656) @mj12albert
+- 📚 Added new guides for integrating with Next.js 13 App Router (#37656) @mj12albert
```

Comment:
I've updated the highlights, do you think we need to directly link to each guide from here? @samuelsycamore

Reply:
Added each guide as a bullet.